### PR TITLE
***WIP*** Add inline code backticks

### PR DESCRIPTION
**Pull request by yuvallanger**
**_WIP**_

### CODE REVIEW

**Comment by terfendail** [comment]
:+1:

**Comment by yuvallanger** [comment]
Shall I add more as I read to this pull request or make many very small pull requests for each encounter?

**Comment by mshabunin** [comment]
@yuvallanger , it is better to create bigger pull requests, but not too big:
- Many small pull requests will create more load on build infrastructure, each change will be tested on multiple platforms.
- Golden mean :sparkles: 
- One big pull request will be reviewed longer, the more files changed the bigger is possibility of conflict with other active pull requests.

You can add `**WIP**` text to the pull request description to mark it _"work in progress"_, then automatic builds will be turned off. After that you can add more changes and discuss them with others until some completion point is reached. Finally someone will approve and integrate it.

File: doc/py_tutorials/py_gui/py_video_display/py_video_display.markdown [hunk 1]
```diff
@@ -55,9 +55,9 @@ video) and full details can be seen here: cv::VideoCapture::get() .
 Some of these values can be modified using **cap.set(propId, value)**. Value is the new value you
 want.
 
-For example, I can check the frame width and height by cap.get(3) and cap.get(4). It gives me
-640x480 by default. But I want to modify it to 320x240. Just use ret = cap.set(3,320) and
-ret = cap.set(4,240).
+For example, I can check the frame width and height by `cap.get(3)` and `cap.get(4)`. It gives me
```
Comment (paroj) [inline doc/py_tutorials/py_gui/py_video_display/py_video_display.markdown]: while at it you could replace 3 and 4 by `cv2.CAP_PROP_FRAME_WIDTH` and `cv2.CAP_PROP_FRAME_HEIGHT`.

**Comment by mshabunin** [comment]
@yuvallanger , is this PR ready for review?

**Comment by mshabunin** [comment]
Replaced: #8787

**Pull request closed without merging**